packages/wrangler/src/tail/createTail.ts

```diff
@@ -186,7 +186,7 @@ export async function createTail(
 	const tail = new WebSocket(websocketUrl, TRACE_VERSION, {
 		headers: {
 			"Sec-WebSocket-Protocol": TRACE_VERSION, // needs to be `trace-v1` to be accepted
-			"User-Agent": `wrangler-js/${packageVersion}`,
+			"User-Agent": `wrangler/${packageVersion}`,
```

Comment:
Any context on this change, other than it looks neater not to have the `-js` postfix? Is it necessary? I assume it don't break anything, but will it affect any reporting tools?

Reply:
Every other request in Wrangler uses the `wrangler/*` user agent, so I'm just making it consistent.